src/list-grid-view-button.js

```diff
@@ -0,0 +1,18 @@
+const listGridButton = document.querySelector(
+  ".list-grid-button[data-view]",
+);
+
+listGridButton.onclick = () => {
```

Comment:
Interesting choice of using attributes to set button state but the major disadvantage to doing this is loosing state on refresh. As such I would recommend using local or session storage to leverage browser APIs to keep state even after refresh.
https://rxdb.info/articles/localstorage.html